assistants/signInAssistant.js

```diff
@@ -0,0 +1,10 @@
+import { PAGES } from "../pages/pages";
+
+export class SignInAssistant {
+  async signInToSite(page, name, password) {
+    await page.locator(PAGES.mainMenu.signInMenu).click();
+    await page.locator(PAGES.signInPopUp.signInUserNameInput).fill(name);
+    await page.locator(PAGES.signInPopUp.signInPasswordInput).fill(password);
+    await page.locator(PAGES.signInPopUp.signUpButton).click();
+  }
```

Comment:
```suggestion
export class SignInAssistant {
  async signInToSite(page, name, password) {
    await page.locator(PAGES.mainMenu.signInMenu).click();
    await page.locator(PAGES.signInPopUp.signInUserNameInput).fill(name);
    await page.locator(PAGES.signInPopUp.signInPasswordInput).fill(password);
    await page.locator(PAGES.signInPopUp.signUpButton).click();
  }
```

you have loginAssistant for this case..

Reply:
but they are different popups with their own locators.  how can I use it?

Reply:
yep. you are right. My bad!